managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -268,6 +286,10 @@ public void triggerComplete() {
             }
         }
 
+        public void alignAcknowledgeStatus() {
+            this.alignAcknowledgeStatusAfterPersisted.run();
+        }
```

Comment:
need to check if the `alignAcknowledgeStatusAfterPersisted != null`

Reply:
It is a final variable, and always not be null. See the constructor method here https://github.com/apache/pulsar/pull/25047/files#diff-fad355f91bd15cc041161f9a46fce62b7fee87fbfb8f0ff8a8b724a1bd1f29eeR257